blocks/graph.py

```diff
@@ -354,3 +386,45 @@ def apply_noise(computation_graph, variables, level, seed=None):
         replace[variable] = (variable +
                              rng.normal(variable.shape, std=level))
     return computation_graph.replace(replace)
+
+
+def apply_dropout(computation_graph, variables, drop_prob=0.5, rng=None,
+                  seed=None):
+    """Returns a graph to variables in a computational graph.
+
+    Parameters
+    ----------
+    computation_graph : instance of :class:`ComputationGraph`
```

Comment:
I can swear we discussed this before (for `apply_noise`) and settled on `ComputationGraph`. Can't remember why, but seems sensible to me.